qdrant-landing/content/documentation/guides/distributed_deployment.md

```diff
@@ -1068,7 +1071,14 @@ await client.CreateCollectionAsync(
 );
 ```
 
-Write operations will fail if the number of active replicas is less than the `write_consistency_factor`.
+Write operations will fail if the number of active replicas is less than the
+`write_consistency_factor`. In this case, the client is expected to send the
+operation again to ensure a consistent state is reached.
+
+Setting the `write_consistency_factor` to a lower value may allow accepting
+writes even if there are unresponsive nodes. Unresponsive nodes are marked as
```

Comment:
If a write is accepted by one node, and that node restarts before it can propagate the write, will the write be lost?